### PR TITLE
ZENKO-949 bugfix: increase default prometheus replica count

### PR DESCRIPTION
**Pull request by giacope**
Increase the default number of Prometheus replicas to 2 for failover purposes.

### CODE REVIEW

**Review by ssalaues — APPROVED** [review]

File: charts/zenko/values.yaml [hunk 1]
```diff
@@ -80,6 +80,8 @@ prometheus:
     enabled: false
   pushgateway:
     enabled: false
+  server:
+    replicaCount: 2
```
Comment (ssalaues) [inline charts/zenko/values.yaml:84]: Does this work without any chart changes?

**Comment by giacope** [comment]
@bert-e wait

File: charts/zenko/values.yaml (hunk 1 above)
Comment (giacope) [reply charts/zenko/values.yaml:84]: No 😥

**Comment by ssalaues** [comment]
Duplicate of #267

**Pull request closed without merging**